### ISSUE TITLE
Spirit Alarms inside ICC

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5914]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The spirit alarms at the trash before marrowgar should be disarmed by rogues but if you try and disarm trap them in stealth nothing will happen, it will only cast disarm trap with no effect at all and deathbound wards will still get triggered when someone steps near the trap
**Expected behaviour**: Tell us what should happen instead
When you use disarm trap as a rogue in stealth you should be able to disarm the spirit alarm so the deathbound wards wont activate if someone steps near it
**Steps to reproduce the problem**:

1. go stealth
2. right click the spirit alarm to disarm trap it
3. see how it will cast disarm trap but nothing will actually happen

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/
https://www.wowhead.com/spell=70547/spirit-alarm
**Include Screenshots from the issue if necessary**
